app/controllers/slack_controller.rb

```diff
@@ -40,22 +40,23 @@ def guidelines
     end
   end
 
-  def auth
-    redirect_to SlackService.get_oauth_url(params[:team_id]).to_s
+  def auth_team
+    redirect_to SlackService.get_team_oauth_url(params[:team_id]).to_s
```

Comment:
You can remove '.to_s' or with other words `get_team_oauth_url` should return a string, so that you do not need to call '.to_s' on that.